proposals/aip-1.md

```diff
@@ -0,0 +1,229 @@
+
+If there are multiple authors, each should be on a separate line following RFC 2822 continuation line conventions. Note that **personal email addresses in AIPs may not be obscured** as a defense against spam harvesters.
+
+While an AIP is a draft, a `discussions-to` header will indicate the URL (preferred) or mailing list where the AIP is being discussed (preferably an issue in [the Issues section of this repository]).
+
+The `type` header specifies the type of AIP: Process or Informational.
+
+The `created` header records the date that the AIP was assigned a number. The header should be in yyyy-mm-dd format, e.g. 2001-08-14.
+
+AIPs may have a `requires` header, indicating the AIP numbers that this AIP depends on.
+
+AIPs may also have a `superseded-by` header indicating that an AIP has been rendered obsolete by a later document; the value is the number of the AIP that replaces the current document. The newer AIP must have a `replaces` header containing the number of the AIP that it rendered obsolete.
+
+Headers that permit lists must separate elements with commas.
+
+
+# Auxiliary Files #
+AIPs may include auxiliary files such as diagrams. Such files must be named `aip-XXXX-Y.ext`, where "XXXX" is the AIP number, "Y" is a serial number (starting at 1), and "ext" is replaced by the actual file extension (e.g. "png").
+
+
+# Transferring Proposal Ownership #
+It occasionally becomes necessary to transfer ownership of AIPs to a new champion. In general, we'd like to retain the original author as a co-author of the transferred AIP, but that's really up to the original author. A good reason to transfer ownership is because the original author no longer has the time or interest in updating it or following through with the AIP process, or has fallen off the face of the 'net (i.e. is unreachable or not responding to email). A bad reason to transfer ownership is because you don't agree with the direction of the AIP. We try to build consensus around an AIP, but if that's not possible, you can always submit a competing AIP.
+
+If you are interested in assuming ownership of an AIP, send a message asking to take over, addressed to both the original author and the AIP editor. If the original author doesn't respond to email in a timely manner, the AIP editor will make a unilateral decision (it's not like such decisions can't be reversed :).
+
+
+# Proposal Editors #
+The current AIP editors are members of the [@Agorex-io/proposal-editors] team.
+
+
+# Proposal Editor Responsibilities and Workflow #
+For each new AIP that comes in, an editor does the following:
+
+- Read the AIP to check if it is ready: sound and complete. The ideas must technically make sense, even if they don't seem likely to be accepted.
+- The title should accurately describe the content.
+- Edit the AIP for language (spelling, grammar, sentence structure, etc.), markup (Github flavored Markdown).
```

Comment:
Should this be the duty of the editor or the submitter?

Reply:
Editors reviews it. They _If the AIP isn't ready, an editor will send it back to the **author** for revision..._ Generally speaking the author will actually make the changes based on feedback from the editor.

Reply:
So then it shouldn't include "Edit the AIP" as a duty of the editor?
Just reviewing the AIP and making suggestions.

Reply:
Yep, thanks. Changing this to _Work with the author/champion to edit the AIP for language..._ I don't think there is a problem if the editor graciously does some cosmetic improvements such as spelling and other housekeeping items to the doc itself. I think it is also fine for the editor to ask the author/champion to make required changes. So my intention is to leave it a little open for interpretation here.